enterprise/releases/2023-06-0.mdx

```diff
@@ -0,0 +1,77 @@
+---
+
+**Docker tag:** `quay.io/pganalyze/enterprise:v2023.06.0`
+
+## Notes
+
+- This release requires a database migration when coming from older releases, be sure to follow the [recommended upgrade steps](/docs/enterprise/upgrade)
+  - **Important**: The included migration on the queries table (20230410170608_add_tables_to_queries) may take considerable time on the pganalyze database when monitoring many database servers. We recommend doing the upgrade during off-hours.
+  - When upgrading, please note that the application will run the following SQL statement - when permissions are restricted, you may need to run this ahead of the upgrade:
+    ```sql
+    CREATE EXTENSION IF NOT EXISTS btree_gin WITH SCHEMA public;
+    ```
+
+## Features
+
+- This release contains preparations for upcoming autovacuum tuning recommendations, as well as the constraint programming based Index Advisor - more on this soon!
+- Wait event monitoring:
+  - Use higher data resolution when viewing a timeframe that's 1 hour or less
+  - Reduce data resolution when viewing longer timeframes to improve clickability
+  - Allow zooming the graph through click+drag, like other graphs in the application
+- Add additional AWS settings for bundled collector
+  - Adds support for configuring the Cluster ID, the Assume Role, as well as allows enabling IAM authentication when using the bundled collector
+- The bundled collector has been updated to [v0.50.0](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md#0500------2023-06-05) (from v0.49.2 in prior release)
+  - This is primarily a bugfix release in the context of an Enterprise Server installation using the bundled collector to monitor a self-managed or
+   Amazon RDS/Aurora database server
+  - In case you have installed the collector separately, we recommend upgrading the collector package/container and reviewing the [full changelog](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md#0500------2023-06-05)
+- Issue e-mail notifications: Include pganalyze organization name in email body, for easier filtering when using multiple organizations
+- Add additional telemetry data to check pganalyze snapshot health via Prometheus/Grafana
+  - See [instructions](/docs/enterprise/settings#worker-queue-monitoring-using-prometheus) for how to use the Prometheus endpoint
+- Allow setting `SIDEKIQ_CONCURRENCY=0` to turn off workers in combined Enterprise image
+  - You can utilize this setting in order to separate the workers from the container servicing the pganalyze web UI, avoiding slow web requests caused by worker processing
+  - Note such a setup requires having configured object storage as well as an external Redis server (set via `REDIS_URL`)
+  - See [all settings](/docs/enterprise/settings)
+
+
+## Performance
+
+- Query analysis: Speed up analysis by avoiding to process schema dependency tracking
+- Improve performance of getIssues by avoiding N+1 queries
```

Comment:
```suggestion
- Improve performance of `getIssues` API endpoint by avoiding N+1 queries
```

I wonder if we can make this `getIssues` more clear that it's an API endpoint, something along that line. At minimum maybe adding backquote?

Reply:
Ah, well I just saw "getIssues API" section in bug fixes. I guess it's okay not to make this as backquote (I also realized that we don't use it much in changelog anyways).
Maybe instead, we say "Improve performance of getIssue API by avoiding N+1 queries"?

Reply:
Good idea on spelling it out - I'll go with " `getIssues` API endpoint" in both places where its mentioned.